core/spec/lib/spree/event_spec.rb

```diff
@@ -18,14 +18,26 @@
     before do
       # ActiveSupport::Notifications does not provide an interface to clean all
       # subscribers at once, so some low level brittle code is required
-      @old_subscribers = notifier.instance_variable_get('@subscribers').dup
+      if Rails.gem_version.segments.first == 6
```

Comment:
Sometimes you use `Rails.gem_version >= Gem::Version.new('6.0.0')` sometimes this, I would prefer to use either the one or the other but not both.

Reply:
@tvdeyen you're absolutely right, I fixed these to always use `Gem::Version.new('6.0.0')` which as far as I can see is the standard way of doing this stuff.